index.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
+  extends: ['tslint-react'],
```

Comment:
@GoodForOneFare Thanks for the feedback. I agree, it's better to break up the repo into multiple configs sooner than later. 

I've created an issue for this: #14